sycl/test-e2e/DeviceArchitecture/device_architecture_on_device_aot.cpp

```diff
@@ -13,7 +13,7 @@ using namespace sycl;
 using namespace sycl::ext::oneapi::experimental;
 
 int main() {
-  std::vector<int> vec(2);
+  std::vector<int> vec(4);
```

Comment:
This might be a bugfix, but not a memory leak fix...

Reply:
Updated